cmd/fyne_demo/main.go

```diff
@@ -238,10 +239,10 @@ func makeNav(setTutorial func(tutorial tutorials.Tutorial), loadPrevious bool) f
 
 	themes := container.NewGridWithColumns(2,
 		widget.NewButton("Dark", func() {
-			a.Settings().SetTheme(&forcedVariant{Theme: theme.DefaultTheme(), variant: theme.VariantDark})
+			a.Settings().SetTheme(test.DarkTheme())
```

Comment:
Hmm, not sure how I feel about importing internal/test in fyne demo. I think in this case the duplicated code of the `forcedVariant` theme is good, since it A) makes fyne_demo depend on nothing but the public APIs, so it is a "true" demo of what is possible for 3rd party apps, and B) does show a code example for a custom theme, which is a common user question now, esp. after the deprecation of DarkTheme and LightTheme